packages/utils/src/common/get-medusa-version.ts

```diff
@@ -0,0 +1,14 @@
+import { join } from "path"
+
+export const getMedusaVersion = (): string => {
+  try {
+    return require(join(
+      process.cwd(),
+      `node_modules`,
+      `@medusajs/medusa`,
+      `package.json`
+    )).version
+  } catch (e) {
+    return ``
```

Comment:
todo: depending on your answer, should we log something at least to get an idea of something that can get wrong?

Reply:
This is a direct copy/paste from what we have in `medusa-core-utils` right now, but definitely something we could add to improve the util.